distro/run/core/src/main/java/org/camunda/bpm/run/property/CamundaBpmRunProperties.java

```diff
@@ -45,6 +45,9 @@ public class CamundaBpmRunProperties {
   @NestedConfigurationProperty
   protected CamundaBpmRunRestProperties rest = new CamundaBpmRunRestProperties();
 
+  @NestedConfigurationProperty
+  protected CamundaBpmRunDeploymentProperties deployment = new CamundaBpmRunDeploymentProperties();
+
   protected CamundaBpmRunAdministratorAuthorizationProperties adminAuth
```

Comment:
❓ Following the [point above](https://github.com/camunda/camunda-bpm-platform/pull/4607/files#r1765122189), shouldn't the `adminAuth` nested properties be also with @NestedConfigurationProperty

Reply:
Let's add it too.